Jenkinsfile

```diff
@@ -1 +1,8 @@
-buildPlugin()
+#!/usr/bin/env groovy
+
+/* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
+buildPlugin(configurations: [
+  [ platform: "linux", jdk: "8", jenkins: null ],
+  [ platform: "windows", jdk: "8", jenkins: null ],
+  [ platform: "linux", jdk: "11", jenkins: "2.150" ]
```

Comment:
2.164.1 please
```suggestion
  [ platform: "linux", jdk: "11", jenkins: "2.164" , javaLevel: "8"]
  [ platform: "windows", jdk: "11", jenkins: "2.164" , javaLevel: "8" ]
```

Reply:
@oleg-nenashev, but 2.164.1 that is not released yet. We need anything that will use java 11.

Reply:
That's why I used 2.164 for now

Reply:
What is the value of using newer JDK and specifying java level from Jenkinsfile like that? Java 8 is properly declared in the pom.